cmd/celestia/internal/admin/admin.go

```diff
@@ -1,69 +1,59 @@
-package main
+package admin
 
 import (
-	"context"
 	"errors"
 	"strings"
 
 	"github.com/filecoin-project/go-jsonrpc/auth"
 	"github.com/spf13/cobra"
+
+	"github.com/celestiaorg/celestia-node/cmd/celestia/internal"
```

Comment:
I'd be in favor of aliasing this to something like `commands`. Internal is useful for restriction in go, but i'm not a fan of it being the package name. It may even make sense to rename things to `cmd/celestia/commands` is using internal wasn't totally deliberate, but i think the rest of the implementation would read much nicer as like `commands.RPCClient` or `commands.PrintOutput` vs `internal.PrintOutput`. Much more descriptive and nicer to read

Reply:
I'm also not a big fan of internal, but in this particular case, I want to hide all cmds from other packages. What if I move all util.go to the separate package and call it `util` -> `util.PrintOutput`? I could also move there a const for `http://localhost:26658` or even instantiate the whole flag for the RequestURL, wdyt @ramin ?

Reply:
yeah i think just alias-ing it and keeping the restriction from internal would be my preference.

on the util idea, i like that same with the const

Reply:
What's the purpose of hiding cmds here?

Reply:
The purpose of hiding is to restrict usage 🙂 .